api/src/main/resources/workbench.yaml

```diff
@@ -2159,10 +2175,58 @@ paths:
           schema:
             $ref: "#/definitions/DataSetPreviewResponse"
 
+  /v1/offline/syncResearcherData:
+    post:
+      tags:
+        - offlineRDRExport
+      description: >
+        Send Researcher data
+      operationId: "syncResearcherData"
+      parameters:
+        - in: body
+          name: researchIds
+          required: true
+          description: The IDs of concepts to be added to the concept set.
+          schema:
+            type: object
+      responses:
+        200:
+          description: >
+            A SQL query for each domain in the Data Set
+
+  /v1/offline/syncWorkspaceData:
+    post:
+      tags:
+        - offlineRDRExport
+      description: >
+        Send Researcher data
+      operationId: "syncWorkspaceData"
+      parameters:
+        - in: body
+          name: researchIds
+          required: true
+          description: The IDs of concepts to be added to the concept set.
+          schema:
+            type: object
+      responses:
+        200:
+          description: >
+            A SQL query for each domain in the Data Set
+
+
 ##########################################################################################
 ## DEFINITIONS
 ##########################################################################################
 definitions:
+  ResearcherId:
```

Comment:
It looks like this definition is unused (unless it's used by the code but not referenced from within the YAML). Remove?